PubNub/Data/PNCBORDecoder.m

```diff
@@ -0,0 +1,879 @@
+/**
+ * @author Serhii Mamontov
```

Comment:
Wow, did you have implement all of this?

Reply:
@bartk yep